modMcf/src/org/aion/mcf/account/FileDateTimeComparator.java

```diff
@@ -76,6 +77,16 @@ public int compare(File arg0, File arg1) {
 
         // TODO: make sure this is the same as the pattern we use to encode
```

Comment:
Is this TODO solved? If so, please remove/edit it.

Reply:
There should be no issues, I will remove it